### ISSUE TITLE
Support windows

### ISSUE THREAD

**Issue by drevell** [abcxyz/abc#70]
### TL;DR

Currently, tests fail on windows. Let's fix them.

The test logs are attached. The failures seem mostly related to file permissions.

[logs_184.zip](https://github.com/abcxyz/abc/files/11901708/logs_184.zip)

Tasks:

 - Investigate each test failure and update either the app logic or test logic to get each one to pass
 - Update test matrix in `.github/workflows` to include windows
 - Update `.goreleaser.yaml` to include windows

How to test on windows: use the [drevell/win-test branch](https://github.com/abcxyz/abc/tree/drevell/win-test) or fork it into your own branch. This branch includes a [workflow that runs go tests on a windows VM](https://github.com/abcxyz/abc/blob/drevell/win-test/.github/workflows/win-test-temporary.yml).

**Comment by pdewilde** [comment]
Can our windows support just print out "please just use WSL"
